Glossary update (#232)
DFE-Digital/bat-design-history: app/glossary.md

Accreditation
: All initial teacher training providers must be ‘accredited’ by the Secretary of State for Education. This process of ‘accreditation’ confirms they have permission to offer courses leading to Qualified Teacher Status (QTS). (NB: Ofsted is the body which checks the courses providers offer meet the standards set by the department.)
: The exception is the School Direct (SD) training route; SDs need a ‘ratifying accredited provider’. The ratifying accredited provider may be a School Centred Initial Teacher Training (SCITT) or a Higher Education Institute (HEI), and is responsible for maintaining the quality of the training provided.

> : As accredited providers, SCITTs can offer QTS courses but do not have ‘degree awarding powers’. So to cater to candidates seeking academic awards (for example, a Post Graduate Certificate in Education (PGCE), they need to partner with an HEI.

Reviewer: That second closing bracket is needed - read the sentence again! Maybe this can be reworded to avoid nested bracketing?

> : As accredited providers, SCITTs can offer QTS courses but do not have ‘degree awarding powers’. So to cater to candidates seeking academic awards (for example, a Post Graduate Certificate in Education (PGCE), they need to partner with an HEI.

Reviewer 2: ```suggestion
: As accredited bodies, SCITTs can offer QTS courses but do not have the power to award a degree. To enable candidates who seek an academic award like a PGCE (Post Graduate Certificate in Education) they need to partner with an HEI.
```

Teaching School Alliance
: Outstanding schools that work with others to provide high quality training to new and experienced staff. Cannot be lead schools or providers.

> Teacher Regulation Agency (TRA)

Reviewer: Ordering should be alphabetical:
* Teacher Analysis Division (TAD)
* Teacher Qualifications Unit (TQU)
* Teacher Reference Number (TRN)
* Teacher Regulation Agency (TRA)
* Teacher supply model (TSM)
* Teaching School Alliance

Author: 🤦‍♀️
